localnet/poktrolld/config/relayminer_config.yaml

```
@@ -1,22 +1,16 @@
 signing_key_name: supplier1
 smt_store_path: smt_stores
+metrics:
+  enabled: true
+  addr: :9090
 pocket_node:
   query_node_rpc_url: tcp://poktroll-validator:36657
   query_node_grpc_url: tcp://poktroll-validator:36658
   tx_node_rpc_url: tcp://poktroll-validator:36657
-proxies:
-  - proxy_name: http-proxy
-    type: http
-    host: 0.0.0.0:8545
 suppliers:
   - service_id: anvil
-    type: http
     service_config:
-      url: http://anvil:8547/
-    proxy_names:
-      - http-proxy
-    hosts:
+      backend_url: http://anvil:8547/
+      publicly_exposed_endpoints:
       - tcp://relayminers:8545
```

I just realized that this should probably just be `relayminers`.
```suggestion
      - relayminers
```

---

1. Probably or definitely?
2. Why?

---

> Probably or definitely

Definitely

> Why

Because the port is specified in the `listen_url` instead.